stable/artifactory-ha/ci/rtsplit-values.yaml

```diff
@@ -63,6 +84,15 @@ jfconnect:
     limits:
       memory: "1Gi"
       cpu: "1"
+  # Add lifecycle hooks for jfconect container
+  lifecycle:
+    postStart:
+      exec:
+        command: ["/bin/sh", "-c", "echo Hello from the jfconnect postStart handler >> /tmp/message"]
+    preStop:
+      exec:
+        command: ["/bin/sh", "-c", "echo Hello from the jfconnect postStart handler >> /tmp/message"]
```

Comment:
I think there is a typo in these messages, they all are "from the jfconnect **postStart** handler" while half of them should be "from the jfconnect **preStop** handler"

Reply:
cc @chukka

Reply:
@lemeurherve Noted , these are just CI files for testing shouldn't impact , will be fixed in next minor release